src/components/inputs/async-select-input/async-select-input.js

```diff
@@ -102,6 +102,7 @@ export class AsyncSelectInput extends React.Component {
     ]),
     loadOptions: PropTypes.func.isRequired,
     cacheOptions: PropTypes.any,
+    shouldDivideOptionGroups: PropTypes.bool,
```

Comment:
I'm not sure about this prop maybe. Maybe `showOptionGroupDivider` would be more appropriate 🤔

Reply:
To me `divide` is already an action, so maybe directly `divideOptionGroups` is enough 😅

Reply:
i think `showOptionGroupDivider` is a little clearer